clifford/_mvarray.py

```diff
@@ -1,6 +1,10 @@
 import numpy as np
 
+import operator
+import functools
 
+from typing import Tuple
```

Comment:
Style is usually to put stdlib imports first:
```suggestion
import functools
import operator
from typing import Tuple

import numpy as np
```